index.js

```diff
@@ -156,5 +164,7 @@ function gql(/* arguments */) {
 gql.default = gql;
 gql.resetCaches = resetCaches;
 gql.disableFragmentWarnings = disableFragmentWarnings;
+gql.print = require('graphql/language/printer');
+gql.parser = parser
```

Comment:
It doesn't look like these two names are consistent - should it be "parse" rather than "parser"?